cpp/ql/lib/semmle/code/cpp/exprs/BuiltInOperations.qll

```diff
@@ -1554,14 +1754,135 @@ class BuiltInBitCast extends BuiltInOperation, @builtinbitcast {
  *
  * Returns `true` if a type is a trivial type.
  * ```
- *  template<typename _Tp>
- *    struct is_trivial
- *    : public integral_constant<bool, __is_trivial(_Tp)>
- *    {};
+ * template<typename _Tp>
+ *   struct is_trivial
+ *   : public integral_constant<bool, __is_trivial(_Tp)>
+ *   {};
  * ```
  */
 class BuiltInIsTrivial extends BuiltInOperation, @istrivialexpr {
   override string toString() { result = "__is_trivial" }
 
   override string getAPrimaryQlClass() { result = "BuiltInIsTrivial" }
 }
+
+/**
+ * A C++ `__reference_constructs_from_temporary` built-in operation
+ * (used by some implementations of the `<type_traits>` header).
+ *
+ * Returns `true` if a type is a trivial type.
+ * ```
+ * template<typename _Tp>
+ *   struct reference_constructs_from_temporary
+ *   : public integral_constant<bool, __reference_constructs_from_temporary(_Tp)>
+ *   {};
+ * ```
+ */
+class BuiltInOperationReferenceConstructsFromTemporary extends BuiltInOperation,
+  @referenceconstructsfromtemporary
+{
+  override string toString() { result = "__reference_constructs_from_temporary" }
+
+  override string getAPrimaryQlClass() {
+    result = "BuiltInOperationReferenceConstructsFromTemporary"
+  }
+}
+
+/**
+ * A C++ `__reference_converts_from_temporary` built-in operation
+ * (used by some implementations of the `<type_traits>` header).
+ *
+ * Returns `true` if a type is a trivial type.
+ * ```
+ * template<typename _Tp>
+ *   struct reference_converts_from_temporary
+ *   : public integral_constant<bool, __reference_converts_from_temporary(_Tp)>
+ *   {};
+ * ```
+ */
+class BuiltInOperationReferenceCovertsFromTemporary extends BuiltInOperation,
+  @referenceconstructsfromtemporary
```

Comment:
This should be `@referenceconvertsfromtemporary` right?

Reply:
Fixed in ed42c3cd6f508e988cbae3e1f0e1139be20087bd